source/sdk/math.ooc

```diff
@@ -59,6 +59,13 @@ floor: extern(floorl) func ~Long (LDouble) -> LDouble
 	 maximumValue ::= static SHRT_MAX
 }
 
+extend Int64 {
+	modulo: func(divisor: This) -> This { 
+		result := this - (this / divisor) * divisor
+		result < 0 ? result + divisor : result
+	}
+}
+
```

Comment:
Okay. I'd *really* like to avoid having these two functions behave differently, but at the same time, modifying the other one here seems risky. Can you make an issue anda top priority to update the one for `Int` as soon as we're done with this merge sprint?